### PR TITLE
Fix/issue121

### PR DESCRIPTION
**Pull request by Jehoel**
Introduces a fix for Issue #121 by generating a Dictionary mapping of filenames to visible solution explorer nodes, instead of assuming nodes always correspond 1:1 with filesystem paths.

Also uses the same logic to enable two new PowerShell commands which either expand or collapse all visible .cshtml's generated.cs files in Solution Explorer: `Show-RazorGeneratedFiles` and `Hide-RazorGeneratedFiles`.

### CODE REVIEW

File: RazorGenerator.Powershell/RazorGenerator.psm1 [hunk 1]
```diff
@@ -29,18 +32,80 @@ function Set-CustomTool {
...
+        [System.Collections.Generic.Dictionary[System.String,EnvDTE.UIHierarchyItem]]$map,
+        [EnvDTE.UIHierarchyItems]$nodes
+    )
+
+    ForEach( $uiHierarchyItem in $nodes ) {
+
+        $projectItem = $uiHierarchyItem.Object # $projectItem.GetType() always returns System.__ComObject
+        If( $projectItem -ne $null ) {
+
+#            Write-Host "$( $projectItem.GetType().FullName )"
```
Comment (pranavkm) [inline RazorGenerator.Powershell/RazorGenerator.psm1]: Could you remove these and \ or make them `Write-Debug` and uncomment them?

File: RazorGenerator.Powershell/RazorGenerator.psm1 [hunk 2]
```diff
@@ -72,6 +147,8 @@ function Enable-RazorGenerator {
         [parameter(ValueFromPipelineByPropertyName = $true)]
         [string]$ProjectName
     )
+
+    Write-Host ("Enable-RazorGenerator")
```
Comment (pranavkm) [inline RazorGenerator.Powershell/RazorGenerator.psm1]: Remove

**Comment by Jehoel** [comment]
I've implemented your feedback and made further improvements. This version should be good-to-go now. I tested it locally and it works.

**Comment by Jehoel** [comment]
Any update?

**Comment by pranavkm** [comment]
> I tested it locally and it works

👍 Fair enough. Merging this in. We can revisit this if people complain. Can you bump up the version number of the vsix (https://github.com/RazorGenerator/RazorGenerator/blob/master/RazorGenerator.Tooling/source.extension.vsixmanifest#L4) to `1.11.1`? That should make it ready to publish to the gallery.

**Comment by Jehoel** [comment]
I've increased the version to `1.11.1` as requested. Should the version in `CommonAssemblyInfo.cs` also be updated?

**Comment by pranavkm** [comment]
Those affect the packages, not the vsix. We can leave that be. Thanks for the PR!

**Comment by JiriCepelkaFirstLineSoftware** [comment]
Issue persist for me. VSIX 1.11.0. RazorGenerator.Mvc 2.4.9.

**Pull request merged**